defining _XOPEN_SOURCE to nothing means the oldest version which means glibc
will not provide the strdup() prototype.  this leads to an implicit decl which
leads to a return type of "int" -- 32bits of a ptr on a 64bit arch leads to
kaboom.

https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=3161

--- text2pcap.c
+++ text2pcap.c
@@ -90,7 +90,7 @@
 #  define __USE_XOPEN
 #endif
 #ifndef _XOPEN_SOURCE
-#  define _XOPEN_SOURCE
+#  define _XOPEN_SOURCE 600
 #endif
 
 #include <ctype.h>
